WISE-Community/WISE-Client
# refactor(Archive Project): Create archive projects button component

## Changes
- Created archive projects button component that the run list and personal project library now use

It looks like there is similar code in ```archive-projects-button.component.ts``` and ```archive-project.service.ts```. We can refactor that in another issue.

## Test
- Make sure archiving/restoring runs and library units using the archive/restore button works
- Make sure archive/restore undo works

Closes #1572
